extensions/slack/src/search.tsx

```diff
@@ -1,11 +1,46 @@
 // This filename should be named `switch-to-channel.tsx` or something similar
 // but it's kept as `search.tsx` as changing the command's name will cause users to lose their keywords and aliases
-import { ActionPanel, Action, Icon, List } from "@raycast/api";
-
+import { ActionPanel, Action, Icon, List, getPreferenceValues } from "@raycast/api";
 import { User, useChannels } from "./shared/client";
 import { withSlackClient } from "./shared/withSlackClient";
 import { useFrecencySorting } from "@raycast/utils";
 import { OpenChannelInSlack, OpenChatInSlack, useSlackApp } from "./shared/OpenInSlack";
+import { convertSlackEmojiToUnicode, getTimeLocale } from "./shared/utils";
+
+const { displayExtraMetadata } = getPreferenceValues<Preferences.Search>();
+
+function getCoworkerTime(coworkerTimeZone: string): string {
+  const time = new Date();
+  return new Intl.DateTimeFormat(getTimeLocale(), {
```

Comment:
Question (non-blocking): Can we not use `date-fns` for this formatting as well? Trying to see if we can remove the time preference altogether.

Reply:
That just might work, @jfkisafk. Removed the preference and replaced it with a +/- in time that shows hour and minutes if available: 
![search tsx — slack-2024-07-22-00 11 36@2x](https://github.com/user-attachments/assets/68767bc3-919f-413a-8772-c3452291c245)

Adds a little more of a cognitive load on the user but I think it's worth it

Reply:
This looks good to me. Thanks for removing the additional preference.